postcss.config.cjs

```diff
@@ -0,0 +1,6 @@
+module.exports = {
+  plugins: {
+    tailwindcss: {},
```

Comment:
So is Post CSS handling the compilation of Tailwind?

Reply:
This would take care of automatic compilation.

Reply:
I'm going to give this a try, look promising!